synapse/federation/federation_client.py

```diff
@@ -571,7 +573,18 @@ def send_request(destination):
             if "prev_state" not in pdu_dict:
                 pdu_dict["prev_state"] = []
 
-            ev = builder.EventBuilder(pdu_dict)
+            # Strip off the fields that we want to clobber.
```

Comment:
could you add something to the docstring to say that we will do the hashing/signing magic?